docs/ref/models/instances.txt

```diff
@@ -621,6 +621,14 @@ from the DB will get updated. In effect there is an automatic
 ``update_fields`` in this case. If you assign or change any deferred field
 value, the field will be added to the updated fields.
 
+.. note::
```

Comment:
A suggestion in case you aren't mindful of [this note](https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#guidelines-for-restructuredtext-files): `To improve readability, use .. admonition:: Descriptive title rather than .. note::. Use these boxes sparingly.`